Testing/functional/tests/cypress/integration/ui/pages/errors.js

```diff
@@ -4,14 +4,10 @@ describe("Error Pages", () => {
     it("HTTP 401", () => {
         cy.visit("/unauthorized");
         cy.contains("h1", "401");
-        cy.contains("h2", "Unauthorized");
```

Comment:
These pages are pretty static, not sure I would consider these tests fragile.

Reply:
Well, they broke when I fixed a typo.